karma.conf.js

```diff
@@ -9,15 +9,15 @@ module.exports = function (config) {
       require('karma-jasmine'),
       require('karma-chrome-launcher'),
       require('karma-jasmine-html-reporter'),
-      require('karma-coverage-istanbul-reporter'),
+      require('karma-coverage'),
       require('@angular-devkit/build-angular/plugins/karma')
     ],
     client: {
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
-    coverageIstanbulReporter: {
+    coverageReporter: {
       dir: require('path').join(__dirname, './coverage/ngx-translate-extract-marker-app'),
-      reports: ['html', 'lcovonly', 'text-summary'],
+      reporters: ['html', 'lcovonly', 'text-summary'],
```

Comment:
I think this should be like this, according to [documentation](https://github.com/karma-runner/karma-coverage#advanced-multiple-reporters):
```suggestion
      reporters: [
          { type: 'html' },
          { type: 'lcovonly' },
          { type: 'text-summary' },
      ],
```